2021/yearbook.html

```diff
@@ -74,6 +74,12 @@ <h2>2021 Software Development Engineer Bootcamp</h2>
           <figcaption>Precious</figcaption>
         </figure>
       </a>
+    </a>
+    <a href="https://https://kamogelo98.github.io/" target="_blank">
+      <figure>
+        <img src="kamo.jpg" alt="Kamogelo" />
```

Comment:
Please use the picture in the images folder `2021/images/kamo.jpg` so src=`images/kamo.jpg` and delete the file `2021/kamo.jpg`